src/components/BubbleDisplay/BubbleDisplay.js

```diff
@@ -5,7 +5,7 @@ import './BubbleDisplay.scss';
 class BubbleDisplay extends Component {
   static propTypes = {
     /** Map of points @todo custom validator */
-    points: PropTypes.object.isRequired,
+    points: PropTypes.array.isRequired,
```

Comment:
I think we will run into issues with using an array when it comes to updating, will be a lot of array filtering. `Object.values()` can transform the result into the array quite efficiently too, unless I'm missing another limitation

Reply:
see https://github.com/digirati-co-uk/timeliner/pull/56/commits/c5244ff3526eee308784ea5a44890f2c8a04d5e3